src/robotFactory.js

```diff
@@ -1,10 +1,63 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(name, weight, coords, chipVersion) {
+    this.name = name;
+    this.weight = weight;
+    this.coords = coords;
+    this.coords.x = coords.x ? coords.x : 0;
+    this.coords.y = coords.y ? coords.y : 0;
+    this.chipVersion = chipVersion;
+  };
 
-class FlyingRobot {}
+  goForward(step = 1) {
+    this.coords.y += step;
+  };
```

Comment:
Add empty line between methods
```suggestion
  };
  
```